tests/test_batch.py

```diff
@@ -42,6 +45,10 @@ def test_create_batch_and_wait_for_results(self, request_mock):
             == f"{self.sdk._client.endpoints.core}/api/v1/jobs/{self.job_id}"
         )
         assert batch.jobs[self.job_id].result == self.job_result
+        job = {"runs": self.n_job_runs, "variables": self.job_variables}
+        batch = self.sdk.create_batch(
+            serialized_sequence=self.pulser_sequence, jobs=[job], wait=True
+        )
```

Comment:
Here what are you trying to test exactly by adding these lines? Maybe we should make a separate test if we are testing another functionality ?

Reply:
Ops, forgot it. It should not be there. I was trying to find a way for mocking a second batch ( with `id` = 2 and different configuration), but I realized that it is not so straightforward.